sdk/dataprotection/Azure.ResourceManager.DataProtectionBackup/api/Azure.ResourceManager.DataProtectionBackup.netstandard2.0.cs

```diff
@@ -900,6 +922,31 @@ public partial class CopyOnExpirySetting : Azure.ResourceManager.DataProtectionB
     {
         public CopyOnExpirySetting() { }
     }
+    public partial class CrossRegionRestoreDetails
+    {
+        public CrossRegionRestoreDetails(string sourceRegion, string sourceBackupInstanceId) { }
+        public string SourceBackupInstanceId { get { throw null; } }
```

Comment:
this looks like an arm resource. Could you please confirm if this is always an arm resource id?
If so, please change the format of this property to `arm-id`